src/client/components/note.vue

```diff
@@ -252,6 +284,10 @@ export default defineComponent({
 		);
 		this.muted = await checkWordMute(this.appearNote, this.$i, this.$store.state.mutedWords);
 
+		(this.tips as any) = await os.api('wallet/tips', { noteId: this.appearNote.id }).catch((e: Error) => {
```

Comment:
Need to make this fetch not query the API so hard or at all.  Currently it has to hit `api/wallet/tips` for each post that is loaded. :disappointed: 
It should instead load the data needed with each post.. :thinking: